ISSUE_TEMPLATE/1-bug-report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: "🐛 Bug report"
+about: "Report a bug with this project"
+title:
+labels: "bug"
+assignees:
+---
+
+## Expected Behavior
+
+Describe what should happen.
+
+## Actual Behavior
+
+Describe what actually happens.
+
+## Step-by-step reproduction instructions
+
+1. Go to '...'
+2. Click on '...'
+3. Scroll down to '...'
+
+## Screenshots, screen recordings, code snippets
+
+If possible, please upload a screenshot or screen recording which demonstrates the bug.
+
+## System Information
+
+- Device: <!--  e.g. iPhone 12 -->
+- Operating System: <!-- e.g. iOS 16.6 -->
+- Browser: <!-- e.g. Chrome 118 -->
+<!-- Or paste a link from https://www.whatsmybrowser.org -->
+- WordPress version: <!-- e.g. "5.8.0". Find this in Tools → Site Health → Info → WordPress -->
+- PHP version: <!-- e.g. "8.2". Find this in Tools → Site Health → Info → Server >
```

Comment:
Closing comment is incorrect here - `>` instead of `-->`

Reply:
resolved, thanks!